sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/SQLViewSuite.scala

```diff
@@ -69,18 +100,105 @@ class SQLViewSuite extends QueryTest with SQLTestUtils with TestHiveSingleton {
     }
   }
 
+  test("correctly parse CREATE TEMPORARY VIEW statement") {
+    withSQLConf(SQLConf.NATIVE_VIEW.key -> "true") {
```

Comment:
This line can be removed now since we no longer support non native view. If there still a config option for native view, can you remove that as well?